h_periodic/h_beat.py

```diff
@@ -17,6 +17,9 @@
     print("h_beat disabled by DISABLE_H_BEAT environment variable")
     sys.exit()
 
+SLIM_TASK_SINCE = "2016-01-01"
+SLIM_TASK_UNTIL = "2018-03-31"
```

Comment:
Include a few more months worth of annotations in the window over the default value.